yarn.lock

```diff
@@ -6193,6 +6193,13 @@
   dependencies:
     tslib "^1.10.0"
 
+"@upstash/redis@^1.18.0":
```

Comment:
are other packages not using these dependencies. Why does charting only need these libraries.

Reply:
This is a needed dependency for the storage adaptor being used from monosize-storage-upstash 
![image](https://github.com/microsoft/fluentui/assets/35366351/231f8934-7a2c-42a3-9008-912745ff5bc9)

Reply:
why dont other packages in fluent need this.

Reply:
Switching to this then